.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,11 +9,6 @@ Checklist
 * [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
 * [ ] Ensured the license file is being packaged.
 
-### Migration
-<!--
-If this is an ABI incompatible change which requries a migration, please note it and ping ``@conda-forge/bot``.
--->
-
```

Comment:
Can you revert this change?

Reply:
@isuruf, that change was made by `conda smithy`, so I've just removed that entire commit.